network/wsNetwork.go

```diff
@@ -1162,16 +1161,19 @@ func (wn *WebsocketNetwork) meshThread() {
 		}
 
 		// TODO: only do DNS fetch every N seconds? Honor DNS TTL? Trust DNS library we're using to handle caching and TTL?
-		dnsAddrs := wn.getDNSAddrs()
-		if len(dnsAddrs) > 0 {
-			wn.log.Debugf("got %d dns addrs, %#v", len(dnsAddrs), dnsAddrs[:imin(5, len(dnsAddrs))])
-			wn.dnsPhonebook.ReplacePeerList(dnsAddrs)
-			mp, ok := wn.phonebook.(*MultiPhonebook)
-			if ok {
-				mp.AddPhonebook(&wn.dnsPhonebook)
+		dnsBootstrapArray := wn.config.DNSBootstrapArray(wn.NetworkID)
+		multiPhonebook := MakeMultiPhonebook()
```

Comment:
it looks like nothing consumes any data gathered into multiPhonebook

Reply:
Yeah - i was just noticing that too.  I think this should be mp.phonebookMap -- we shouldn't have a local multiPhonebook instance.

Reply:
good catch, fixed.